lib/PatchNavigation.mli

```diff
@@ -0,0 +1,24 @@
+(** This module defines navigation and summary functions for viewing changes in a diff viewer application using the Nottui library. It supports navigating between patches and displaying summaries of changes. *)
+
+type direction =
+  | Prev
+  | Next
+      (** Type representing navigation directions within a list of patches. 
+          - [Prev] indicates navigation to the previous item.
+          - [Next] indicates navigation to the next item. *)
+
+val quit : bool Lwd.var
+(** [quit] is a variable indicating whether the application should terminate its main loop. *)
+
+val navigate : 'a Zipper.t Lwd.var -> direction -> unit
+(** [navigate z_patches dir] navigates within a zipper of patches in the specified direction [dir].
+    It updates the zipper's state to the previous or next item based on the direction.
+    @param z_patches A reactive variable containing a zipper of patches.
+    @param dir The direction to navigate within the zipper.
+    @return () Updates the state of [z_patches] without returning any value. *)
+
+val change_summary : Patch.t Zipper.t Lwd.var -> Nottui.ui Lwd.t
```

Comment:
This does not seem to be related with patch navigation. Either it should be moved, or the name of this module should be more general!

Alsto, this should not take a `'a Zipper.t Lwd.t`: the change summary has no reason to modify the variable!
I think the best signature for this would be: `Patch.t Zipper.t -> Nottui.ui`